lib/miq_automation_engine/engine/miq_ae_service/miq_ae_service_rbac.rb

```diff
@@ -0,0 +1,37 @@
+module MiqAeMethodService
+  module MiqAeServiceRbac
+    extend ActiveSupport::Concern
+
+    module ClassMethods
+      def find_ar_object_by_id(id)
+        rbac = Thread.current.thread_variable_get(:miq_rbac)
```

Comment:
wrap this variable in a `rbac_enabled?` `rbac_enabled=` (or `enable_rbac` / `disable_rbac`)